src/ui/__main__.py

```diff
@@ -1,5 +1,5 @@
 # noqa: D100
-from .run import serve
+from .run import serve_debug
 
 if __name__ == "__main__":
```

Comment:
Inside `__main__.py` this I believe is redundant.

Ref.: https://docs.python.org/3/library/__main__.html

Reply:
Mentioned this in another thread, but i dont wanna change UI main too much since this is a pull request for the config package. It should be changed in its own pull request concerning refactoring UI.